design-toolkit/components/src/components/action-bar/styles.ts

```diff
@@ -10,7 +10,7 @@
  * governing permissions and limitations under the License.
  */
 
-import { tv } from 'tailwind-variants';
+import { tv } from '@/lib/utils';
```

Comment:
cc @switzerb @ArrayKnight wanted to make sure this change is okay. I noticed quite a few components were using `tailwind-variants` directly instead of our `@/lib/utils` `tv` wrapper.

Reply:
If this is intentional I can revert!

Reply:
This is a good catch! We definitely want to be using our internal tv function